src/contexts/WalletSelectorContext.tsx

```diff
@@ -179,8 +180,7 @@ export const WalletSelectorContextProvider: React.FC<{
 
   useEffect(() => {
     init().catch((err) => {
-      console.error(err);
-      alert("Failed to initialise wallet selector");
+      console.error("Failed to initialise wallet selector", err);
```

Comment:
Maybe better to log a single error object still. Not sure how this is going to come through in Sentry (and will look a bit odd in the console). Could modify the message, something like:

```suggestion
      console.error(new Error(`Failed to initialise wallet selector: ${err.message}`));
```